starknet_py/contract.py

```diff
@@ -220,35 +235,32 @@ async def estimate_fee(
                 "Cannot estimate fee of PreparedFunctionCall with max_fee not None or 0."
             )
 
-        tx = await self._account_client.sign_invoke_transaction(
+        tx = await self._account.sign_invoke_transaction(
             calls=self, max_fee=0, version=self.version
         )
 
-        return await self._account_client.estimate_fee(
+        return await self._client.estimate_fee(
             tx=tx, block_hash=block_hash, block_number=block_number
         )
 
-    @property
-    def _account_client(self) -> AccountClient:
```

Comment:
This is breaking. We should remember about it in the docs

Reply:
Agreed

There's couple of different breaking changes like the change in the constructors of `ContractFunction` but these aren't probably created by the user.

Reply:
This is a private property.

Reply:
Oh, you're right. So we don't have to worry about it at all 😅